### PR TITLE
Update repository to be truly commonjs compatible

### PR DESCRIPTION
**Pull request by shrishrirang**
- Closes https://github.com/Azure/azure-mobile-apps-js-client/issues/88, https://github.com/Azure/azure-mobile-apps-js-client/issues/130 and https://github.com/Azure/azure-mobile-apps-js-client/issues/158

### CODE REVIEW

**Comment by shrishrirang** [comment]
@danderson00, @jvano, @adrianhall

File: sdk/src/Platform/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+if (window && window.cordova && window.cordova.version) {
```
Comment (danderson00) [inline sdk/src/Platform/index.js]: This should be if(typeof window !== 'undefined' && ...

**Comment by danderson00** [comment]
Looks good, simplifies the grunt configuration a fair bit, and bye bye WinJS!

File: sdk/src/Platform/index.js (hunk 1 above)
Comment (shrishrirang) [inline sdk/src/Platform/index.js]: Yes, I realized that. Didn't bother fixing it because this is going to change in the very next PR.

**Pull request merged**